#!/usr/bin/env python

PACKAGE = 'jsk_perception'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#       name    type     level     description     default      min      max
gen.add("nms_thresh", double_t, 0, "Threshold for non maximum suppression", 0.45, 0.0, 1.0)
gen.add("score_thresh", double_t, 0, "Threshold for confidence score", 0.60, 0.0, 1.0)
gen.add("profiling", bool_t, 0, "Enable profiling process time", False)

exit(gen.generate(PACKAGE, PACKAGE, "SSDObjectDetector"))
